probe-builder/build-probe-binaries

```diff
@@ -0,0 +1,869 @@
+		$DIR/oracle-kernel-crawler.py OL6-UEK | xargs -n 1 -P $DOWNLOAD_CONCURRENCY wget -c -nv --timeout=${DOWNLOAD_TIMEOUT} --tries=${RETRIES} -P $DISTRO
+	fi
+	echo "Building Oracle Linux 6 UEK"
+	rhel_build $DISTRO/*.rpm
+}
+
+function build_group_oracle_ol7_uek {
+	#
+	# Oracle Linux 7 UEK build
+	#
+	DISTRO=oracle-uek7
+	BUILDER_DISTRO=oracle-uek7
+	DOCKER_BUILDER=ol7
+	if [ "$DOWNLOAD_CONCURRENCY" != "0" ]
+	then
+		echo Downloading Oracle Linux 7 UEK kernels
+		DIR=$(dirname $(readlink -f $0))
+		mkdir -p $DISTRO
+		$DIR/oracle-kernel-crawler.py OL7-UEK | xargs -n 1 -P $DOWNLOAD_CONCURRENCY wget -c -nv --timeout=${DOWNLOAD_TIMEOUT} --tries=${RETRIES} -P $DISTRO
+	fi
+	echo "Building Oracle Linux 7 UEK"
+	rhel_build $DISTRO/*.rpm
+}
+
+function build_group_amazonlinux {
+	#
+	# AmazonLinux build
+	#
+	DISTRO=amazonlinux
+	if [ "$DOWNLOAD_CONCURRENCY" != "0" ]
+	then
+		echo Downloading AmazonLinux kernels
+		DIR=$(dirname $(readlink -f $0))
+		mkdir -p $DISTRO
+		$DIR/kernel-crawler.py AmazonLinux | xargs -n 1 -P $DOWNLOAD_CONCURRENCY wget -c -nv --timeout=${DOWNLOAD_TIMEOUT} --tries=${RETRIES} -P $DISTRO
+	fi
+	echo "Building AmazonLinux"
+	rhel_build $DISTRO/*.rpm
+}
+
+function build_group_amazonlinux2 {
+	#
+	# AmazonLinux build
+	#
+	DISTRO=amazonlinux2
+	if [ "$DOWNLOAD_CONCURRENCY" != "0" ]
+	then
+		echo Downloading AmazonLinux2 kernels
+		DIR=$(dirname $(readlink -f $0))
+		mkdir -p $DISTRO
+		$DIR/kernel-crawler.py AmazonLinux2 | xargs -n 1 -P $DOWNLOAD_CONCURRENCY wget -c -nv --timeout=${DOWNLOAD_TIMEOUT} --tries=${RETRIES} -P $DISTRO
+	fi
+	echo "Building AmazonLinux2"
+	rhel_build $DISTRO/*.rpm
+}
+
+function build_group_rhel {
+	if [ -z "$ARTIFACTORY_KEY" -o -z "$ARTIFACTORY_SERVER" ]
+	then
+		echo "Artifactory credentials missing, skipping RHEL build"
+		return
+	fi
+
+	#
+	# RHEL build from Stored Sources
+	#
+	DISTRO=rhel
+	if [ "$DOWNLOAD_CONCURRENCY" != "0" ]
+	then
+		echo Downloading RHEL kernels
+		mkdir -p $DISTRO
+
+		# We're using aql to query for all of the objects in the repo
+		# and jq to parse the json to get the rpms
+		curl -s -H 'Content-Type: text/plain' -H "X-JFrog-Art-Api: $ARTIFACTORY_KEY" -X POST \
+			-d 'items.find({"repo":"redhat-sources"})' https://$ARTIFACTORY_SERVER/artifactory/api/search/aql | \
+			jq -r '.results[].name as $o | $o | select(endswith(".rpm")) | $o' | \
+			xargs -n 1 -P $DOWNLOAD_CONCURRENCY -iRPM wget -c -nv --timeout=${DOWNLOAD_TIMEOUT} --tries=${RETRIES} -P $DISTRO https://$ARTIFACTORY_SERVER/artifactory/redhat-sources/RPM --header "X-JFrog-Art-Api: $ARTIFACTORY_KEY"
+	fi
+
+	echo Building RHEL from Stored Sources
+	rhel_build $DISTRO/*.rpm
+}
+
+function build_everything {
+	build_group_ubuntu
+	build_group_centos
+	build_group_fedora
+	build_group_fedora_atomic
+	build_group_coreos
+	build_group_debian
+	build_group_oracle_rhck
+	build_group_oracle_ol6_uek
+	build_group_oracle_ol7_uek
+	build_group_amazonlinux
+	build_group_amazonlinux2
+}
+
+case "$KERNEL_TYPE" in
```

Comment:
Suggest you add a comment here to let someone who might add a new kernel type to also update the usage message above.

Reply:
I added it at the bottom of the case block instead (where you'll probably start adding new stuff)